.azure-pipelines/jobs/test-windows.yml

```diff
@@ -32,6 +32,9 @@ jobs:
     vmImage: ${{ parameters.vmImage }}
   strategy:
     matrix:
+      Python35-x64:
```

Comment:
We don't want to run integration tests for this platform on Azure? I don't know what criteria we use to pick integration test platforms. If we do want to run integration tests then I think it would be appropriate to just move this to the Test_Primary job vs keeping this in secondary and setting `runIntegrationTests`. Otherwise we'll have 2 sets of integration tests running in sequence.

Reply:
Good point :+1: , I overlooked the `runIntegrationTests` part. I moved Python35-x64 to Primary.